src/olympia/search/filters.py

```diff
@@ -736,19 +736,24 @@ def apply_search_query(self, search_query, qs, sort=None):
                     Q('term', is_disabled=False)
                 )
             }),
-            query.SF({
-                'weight': 5.0,
-                'filter': (
-                    Q('term', is_recommended=True)
-                )
-            }),
         ]
+        if switch_is_active('api-recommendations-priority'):
+            recommendation_functions = [
+                query.SF({
+                    'weight': 5.0,
+                    'filter': (
+                        Q('term', is_recommended=True)
+                    )
+                }),
+            ]
+        else:
+            recommendation_functions = []
 
         # Assemble everything together
         qs = qs.query(
             'function_score',
             query=query.Bool(should=primary_should + secondary_should),
-            functions=functions)
+            functions=functions + recommendation_functions)
```

Comment:
Instead of this, you could just do `functions.append(...)` after the `if switch_is_active` maybe?

Reply:
I thought it was clearer to use `+` but can do

Reply:
https://github.com/mozilla/addons-server/pull/11436/commits/53e9f1a467932369904c5256cac664d79490558b